README.md

```diff
@@ -18,16 +18,16 @@ This role requires the [php-fpm role](https://github.com/stuvusIT/php-fpm), the
 | `wordpress_default_user`     | `admin`            | This is the user that is created on setup                                     |
 | `wordpress_default_password` | :heavy_check_mark: | The password that is used for the default user                                |
 | `wordpress_default_email`    | :heavy_check_mark: | The email address for the default user                                        |
-| `wordpress_plugin_merge`     | `true`             | The email address for the default user                                        |
+| `wordpress_plugin_merge`     | `true`             | Default behaviour when using plugin options                                   |
```

Comment:
Just use the same description as below and rename the variable to include `_default`, then the behaviour is clear.